src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -41,7 +41,7 @@ public class List<T> : IList<T>, System.Collections.IList, IReadOnlyList<T>
         [NonSerialized]
         private Object _syncRoot;
         
-        static readonly T[]  _emptyArray = new T[0];        
+        static readonly T[]  _emptyArray = Array.Empty<T>();        
```

Comment:
It's not clear that this is a win.  If the only ```T```s ever used with ```List<T>``` were ones that were being used with ```Array.Empty<T>()``` elsewhere, then this would make sense, in that we avoid an extra ```T[0]```.  But if ```List<T>``` is used with a ```T``` that's not otherwise being used with ```Array.Empty<T>```, then we're now forcing ```Array.Empty<T>``` to do duplicative caching (and the associated codegen costs) that it otherwise wouldn't need to do.  This is one we may want to revisit in the future, especially if we can verify that all of the relevant backends will do the right thing regarding getting rid of the _emptyArray static field entirely and just using ```Array.Empty<T>``` at the call sites, but for now let's revert this.